src/App.tsx

```diff
@@ -55,7 +55,7 @@ const App = (props: AppProps) => {
                         <Heading spacing size="small" level="3">
                             Testside for Overblikk
                         </Heading>
-                        <BodyShort>
+                        <BodyShort tabIndex={2}>
```

Comment:
🤔

Reply:
Brukes kun for å dobbelsjekke at eslint faktisk kjører, skal fjernes nå etterpå! Denne branchen er ennå i draft modus